cpp/ps0003/ps0003_soin_cpp.cpp

```diff
@@ -2,11 +2,15 @@
 #include<conio.h>
 using namespace std;
 int main(){
+	// prompting user for number 2/2
 	int number;
 	cin >> number;
+
+	// checking if the number is odd or even 3/3
 	if (number % 2 == 0)
-		cout << "Number is evan\n";
+		cout << "Number is even\n";
 	else
 		cout << "Number is odd\n";
 	_getch();
-}
\ No newline at end of file
+}
+// great work 😎 9/10 ⭐

```

Comment:
- [x] program returns `0` as everything is okay
- [x] program is in the correct path
- [x] no syntax errors
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code
you can visit the [problem solution](https://onlinegdb.com/PbqxNbSVc) for more

you also can use _ternary operator_:
```cpp
std::cout << "Number is " << ((number % 2 == 0) ? "even" : "odd") << std::endl
```